src/plugins/vis_augmenter/public/saved_augment_vis/utils/test_helpers.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { cloneDeep } from 'lodash';
+import { VisLayerExpressionFn, ISavedAugmentVis } from '../../types';
+
+const pluginResourceId = 'test-plugin-resource-id';
+const visName = 'visualization_0';
+const visId = 'test-vis-id';
+const version = 1;
+
+export const generateAugmentVisSavedObject = (idArg: string, exprFnArg: VisLayerExpressionFn) => {
+  return {
+    id: idArg,
+    pluginResourceId,
+    visLayerExpressionFn: exprFnArg,
+    visName,
+    visId,
+    version,
+  } as ISavedAugmentVis;
+};
+
+export const getMockAugmentVisSavedObjectClient = (
```

Comment:
out of curiosity - was this a pattern you picked up from elsewhere, or something new you decided would be useful?

Reply:
I saw roughly similar patterns for existing saved object loaders - example: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/saved_objects/public/saved_object/saved_object_loader.test.ts#L23-L28